ui/ui.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 type Model struct {
-	keys          keys.KeyMap
+	keys          *keys.KeyMap
```

Comment:
I am actually not sure this is used, it should be a duplicate of the pre-created KeyMap from the keys.go file.